roles/nginx_base/tasks/ssl_reject_handshake.yml

```
@@ -0,0 +1,15 @@
+---
+- name: Install ssl-reject-handshake host
+  ansible.builtin.copy:
+    src: ssl-reject-handshake
+    dest: /etc/nginx/sites-available/
```

```suggestion
    dest: /etc/nginx/sites-available/default-ssl-reject-handshake
```

Maybe add "Default" to the name of the file somewhere.